chat-client/README.md

```diff
@@ -1,3 +1,14 @@
 # Q Chat Client
 
-This package contains Q Chat Client that can be used with LSP Chat language server. The client is based on [MynahUI] (https://github.com/aws/mynah-ui).
+This package contains a chat client for the Q Language Server, that can be embedded in a webview. It renders the [MynahUI](https://github.com/aws/mynah-ui) library inside the webview.
```

Comment:
nit: This library does not render MynahUI inside the webview, so this sentence can be confusing.

Reply:
'It uses MynahUI to render a web based chat interface' or sth like that maybe a bit more relevant